web/client/utils/CoordinatesUtils.js

```diff
@@ -1020,6 +1020,22 @@ export const makeBboxFromOWS = (lcOWS, ucOWS) => {
     return [lc[0], lc[1], uc[0], uc[1]];
 };
 
+/**
+ * helper use to transform extent to bound object { minx, miny, maxx, maxy }
+ * if there is aprovided param extent it will return the default bound object of wgs84
+ * @param {extent} extent is an array of 4 ordered coordinates [minx, miny, maxx, maxy]
```

Comment:
you can use [this](https://jsdoc.app/tags-param.html) technique 
```suggestion
 * @param {number[]} extent is an array of 4 ordered coordinates [minx, miny, maxx, maxy]
```